app/graphql/mutations/watch_transaction_mutation.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module Mutations
+  class WatchTransactionMutation < Types::Base::BaseMutation
+    description 'Given a transaction, save it to the database to be resent'
+
+    argument :transaction_object, GraphQL::Types::JSON,
+             required: true,
+             description: 'The JSONified transaction data object'
+    argument :signed_transaction, String,
+             required: true,
+             description: 'Signed transaction in HEX format'
```

Comment:
Issue description states only these fields in the mutation. Which means that transaction_object should be object for sendTransaction merged with receipt.
Shall I add it?

Reply:
Just confirmed with @mandres-digix  that the `txhash` is in the `transactionObject`, so we can use that instead.

Reply:
Also update the  `description` for `transactionObject` that it  requires a `txhash` and `nonce` field.

Reply:
@tymat told me: "An small oversight on my part is using the txhash inside the transaction_object as part of the requirement when creating a watched transaction. This is a small change so that the frontend can resend the transaction given the known txhash."
Couldn't confirm with him yet, but I guess that means that we should have txhash outside of the transactionObject.
What @mandres-digix told you? txhash would be in the transactionObject only if you merge transaction data object with transaction receipt object.

Reply:
Okay, I checked the `spectrum` code myself and the you are correct with the receipt not being inside the transaction object. Sorry about that, I guess we're going to add an tiny field `txhash` for watching transactions. 

Going to update the document with the `txhash` for watching transactions